src/cli.c

```diff
@@ -43,6 +43,7 @@ void print_help(void)
     SDL_Log("  %-*s %s", column_width, "-t | --stage <value>", "Shader stage. May be inferred from the filename. Values: [vertex, fragment, compute]");
     SDL_Log("  %-*s %s", column_width, "-e | --entrypoint <value>", "Entrypoint function name. Default: \"main\".");
     SDL_Log("  %-*s %s", column_width, "-m | --shadermodel <value>", "HLSL Shader Model. Only used with HLSL destination. Values: [5.0, 6.0]");
+    SDL_Log("  %-*s %s", column_width, "-i | --include <value>", "HLSL include directory. Only used with HLSL source. Optional.");
```

Comment:
Would `-I` make more sense?

Reply:
That's what DXC does so yeah let's go with that.